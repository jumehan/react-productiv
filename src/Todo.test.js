import Todo from "./Todo";
import { render } from "@testing-library/react";

let todo = {
  id:3,
  title:"Go to bed",
  description:"In bed by 11:15",
  priority:3}

it("renders without crashing", function () {
  render(<Todo todo = {todo} />);
});

it("matches snapshot", function () {
  const { asFragment } = render(<Todo
    todo = {todo}/>);
  expect(asFragment()).toMatchSnapshot();
});

