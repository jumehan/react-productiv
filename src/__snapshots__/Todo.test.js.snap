// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Todo"
    id="3"
  >
    <div>
      <b>
        Go to bed
      </b>
       
      <small>
        (priority: 3)
      </small>
    </div>
    <div>
      <small>
        In bed by 11:15
      </small>
    </div>
  </div>
</DocumentFragment>
`;
